src/frontend/komponenter/Felleskomponenter/Datovelger.tsx

```diff
@@ -86,6 +86,10 @@ const Datovelger = ({
         },
     });
 
+    useEffect(() => {
+        setSelected(felt.verdi);
```

Comment:
Hvorfor må man gjøre dette manuelt? 🤔

Reply:
Hvis skjemafeltet først blir satt til undefined, og millisekundet senere får en verdi (som feks hentes fra backend), så er det undefined som datovelgeren setter som sin verdi, fordi man kun detter `defaultSelected.` Med denne useeffecten sørger vi for at det alltid er den faktiske verdien til feltet som rendres av komponenten!